som

```diff
@@ -224,6 +228,8 @@ if not args.interpreter and args.igv_to_file:
     flags += ['-Dgraal.PrintIdealGraphFile=true']
 if args.igv_methods:
     flags += ['-Dsom.igvDumpAfterParsing=true']
+if args.no_ansi_coloring:
```

Comment:
Why are there two flags added?
`-Dsom.noAnsiColoring=true` and `-Dsom.useANSIColor=false`?

Reply:
Haha, missed this during clean-up.